cli/cmd/integration.go

```diff
@@ -106,12 +102,7 @@ var (
 				return errors.New(msg)
 			}
 
-			cli.OutputHuman(
-				renderSimpleTable(
```

Comment:
This looks like a regression, we need to use the new `renderSimpleTable()` func.